.travis.yml

```
@@ -225,15 +225,13 @@ matrix:
           env: TOXENV="py37-test-devdeps"
                TOXPOSARGS="--remote-data=any"
 
-        # Mysterious failure, cannot find python.exe
-        - os: windows
-          stage: Final tests
-          name: Python 3.7 with all optional dependencies
-          env: PYTHON_VERSION=3.7
-               TOXENV='py37-test-alldeps'
-
 before_install:
 
+    - if [[ $TRAVIS_OS_NAME == windows ]]; then
+        choco install python --version 3.8;
```

```suggestion
        choco install --no-progress python --version 3.8;
```
Otherwise the log is full of "Downloading..." messages